src/main/resources/application.conf

```diff
@@ -105,5 +105,6 @@ userprofile {
 
 nih {
   whitelistFile = "users.txt"
-  rawlsGroupName = "dbGapAuthorizedUsers"
+  rawlsGroupName = "testRealm"
+//  rawlsGroupName = "dbGapAuthorizedUsers"
```

Comment:
That will be the case until it gets re-added in. An admin in former-Blue should be able to get that done.